packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -137,24 +192,37 @@ export default class Meeting extends StatelessSparkPlugin {
   }
 
   /**
+   * // TODO: change name to genertic parser
    * Set the locus info the class instance
    * @param {Object} locus
    * @returns {null}
+   * @memberof Meeting
    */
   setLocusJoin(locus) {
-    this.locus = locus.locus;
+    const mtgLocus = locus.locus || locus;
+    this.locus = mtgLocus;
     this.mediaConnections = locus.mediaConnections;
-    this.locusUrl = locus.locusUrl;
+    this.locusUrl = locus.locusUrl || locus.url;
     this.locusId = locus.locusId;
     this.selfId = locus.selfId;
     this.mediaId = locus.mediaId;
+    this.participants.set(mtgLocus);
+  }
+
+  getLocusType(locusTags) {
+    // Parse all the value and get the type of meeting
+    if (locusTags.find((tag) => tag === 'ONE_ON_ONE')) {
+      return 'ONE_ON_ONE';
+    }
+    return null;
```

Comment:
What's the advantage of returning `null` here rather than `undefined` or an empty string?